lib/config/config-utils.go

```diff
@@ -198,6 +203,10 @@ func (c *Configuration) getVersionInfo() (string, int, *errco.MshLog) {
 		return utility.FirstNon("", info.Version1, info.Version2), info.Protocol, nil
 	}
 
+	if c.Server.FileName != "server.jar" { //check if it matches the deafult name, if not
+		return c.getVersionInfo("server.jar") //fallback to default naming scheme
+	}
+
```

Comment:
@JackGlobetrotter sorry for late reply, I'm super busy irl. Why did you add this check? it seems to me that it makes useless the logic implemented at begin of function.